config/game_options.txt

```diff
@@ -105,30 +105,30 @@ ALERT_DELTA Destruction of the station is imminent. All crew are instructed to o
 
 # New
 PROBABILITY DARKSPAWN 0 # Moved to midround event
-PROBABILITY HERESY 4
+PROBABILITY HERESY 6
 PROBABILITY INFILTRATION 0
-PROBABILITY BLOODSUCKER 4
+PROBABILITY BLOODSUCKER 5
 
 # Lowpop
 PROBABILITY TRAITOR 5
 PROBABILITY TRAITORBRO 5
-PROBABILITY TRAITORCHAN 6
-PROBABILITY TRAITORSUCKER 3
+PROBABILITY TRAITORCHAN 4
+PROBABILITY TRAITORSUCKER 6
 PROBABILITY TRAITORVAMP 4
 PROBABILITY INTERNAL_AFFAIRS 3
-PROBABILITY CHANGELING 4
+PROBABILITY CHANGELING 3
 PROBABILITY VAMPIRE 0
 
 # Group antags
-PROBABILITY REVOLUTION 8
+PROBABILITY REVOLUTION 7
```

Comment:
why reduction in revs?

Reply:
As is revs is effectively the same as blitz ops (round ends quickly, no tells for antagonists) or is massively delayed & unlike nukeops there is no reason to NOT speedrun a rev round as the revs since you are much stronger in ambushes than open combat & don't get fun tools